core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestTracingProcessor.java

```diff
@@ -146,4 +184,177 @@ public void visitCode() {
             };
         }
     }
+
+    @BuildStep
+    ConsoleCommandBuildItem testConsoleCommand(CombinedIndexBuildItem indexBuildItem) {
+        return new ConsoleCommandBuildItem(new TestCommand());
+    }
+
+    @GroupCommandDefinition(name = "test", description = "Test Commands", groupCommands = { TagsCommand.class,
+            PatternCommand.class })
+    public static class TestCommand implements Command {
+
+        @Override
+        public CommandResult execute(CommandInvocation commandInvocation) throws CommandException, InterruptedException {
+            return CommandResult.SUCCESS;
+        }
+    }
+
+    @GroupCommandDefinition(name = "tags", description = "Test Commands", groupCommands = { IncludeTagsCommand.class,
```

Comment:
`"Test Commands"` looks like a c&p mistake.
```suggestion
    @GroupCommandDefinition(name = "tags", description = "Tag Commands", groupCommands = { IncludeTagsCommand.class,
```
@geoand WDYT?

Reply:
Makes sense

Reply:
I'll change it after CI is done.

Reply:
Done.